### DISCUSSION TITLE
How can I get verified for Student Developer Pack?

### DISCUSSION THREAD

**Discussion by JerryGao818** [community/community#51058] [GitHub Education] [answered]
I have tried everything I can do for the verification... I have upload a screenshot of my school portal that clearly shows my school name, enrollment time and expected graduation time. I have update my profile with my full name. All of this just can't work. The same message remains:
![image](https://user-images.githubusercontent.com/81412682/227778422-c9a6ca66-d895-468d-8ed0-88f902d5d7b3.png)

What should I do?

--------------------------------update----------------------------------
Finally I have my verification thanks to all the valuable suggestions!
For someone like me:

I have occasionally changed my browser and when I process my application, there was a message trying to get my location (never seen before). It may be important to keep the location the same as my school.
Using the dated student ID card, as @KleeMatrix mentioned above. I have tried with other document, and they just didn't work.
If the problem remains, please refer to the other answers in this discussion.

对于国内学生，希望我的经历能帮到大家：

记得允许浏览器获取当前位置，确保当前位置和自己的学校位置一致
用学生证的照片，我用的是封面和有注册章的那页，中文没有翻译也没关系，其他的包括教育部学籍在线验证报告之类的我都试过不好用。
保持自己profile里面的各种信息和学生证一致。
如果还有问题，也可以参考这个问题下面的其他回答。
Thanks again for all the kind help! Hope this can help someone like me :)

**Comment by KotlinHero** [comment]
Me too.I'm really at a loss and don't know what document to upload next.

**Reply by KotlinHero** [reply]
Chinese？我也是，各种文件都传了就是过不了，真服了

**Reply by JerryGao818** [reply]
唉 试了下面的方法还是不行

**Comment by xuefengze** [comment]
Me, too. I was rejected many many times.

**Comment by Ntshangase** [comment]
Hello, What I can suggest is that instead of uploading a screenshot of your school portal maybe try uploading proof of registration at that institution, and make sure that your GitHub profile contains a name similar to the one you have on your submitted document

**Comment by suraj719** [comment]
hey, try to add the mail id given by your school to your accout (if given) and instead of attaching the screenshot of your school portal attach an image containing a pysical reciept or identity card of you school. Tx :)

**Comment by AJBrownson** [comment]
I am having the same issue also
I am using my ID card but it's still throwing errors

**Reply by Ntshangase** [reply]
Ensure your profile contains some reference to your evidence. The verification process cross references your GitHub username, email and profile name (and other aspects). Do any of these match on your evidence?

Although Dated School ID is preferred, GitHub does support alternative evidence. Try applying again after updating your GitHub profile

**Reply by AJBrownson** [reply]
So I updated my GitHub profile username to match my school's ID name
But now I'm being told my proof has already been used
What else can I do?

**Comment by JerryGao818** [comment]
Thanks for all the kind helps. I have added my official email id of my school.  I have uploaded an image containing my student status, my school ID card and the screenshot of my school portal. I have updated my profile to make sure the name is consistent to my document. But it still doesn't work. 
![image](https://user-images.githubusercontent.com/81412682/227870686-d0047a72-216d-4a0c-9355-ddc8ecf78b92.png)

Maybe I can go to find some physical receipt of my enrollment. If anyone having the same trouble finally gets the verification, please kindly share the solution. Thank you again for the kind reply. :)

**Reply by GentleM4dman** [reply]
我在申请过程中也遇到了与你相同的问题，多次修改user profile也无果。但最后发现实际应该不是user profile的问题，我改为上传了实体的学生证（中文），成功认证。

**Reply by JerryGao818** [reply]
我试了各种文件包括教育部的在线认证啥的，唯独因为学生证不在手边还没试过，待会回去试一下学生证。非常感谢！

**Reply by Ntshangase** [reply]
If you feel like you have done everything correctly but still getting rejected, I would advise you to go through your application and try find errors, sometimes the little thing can make the application not succeed.

**Reply by JerryGao818** [reply]
> If you feel like you have done everything correctly but still getting rejected, I would advise you to go through your application and try find errors, sometimes the little thing can make the application not succeed.

I will double-check my application and try with my dated physical receipt to see if it can work. Thank you very much for your patience!

**Comment by ivan-aksamentov** [comment]
@JerryGao818 They say that you need to write your full name as it appears on your university document. In your profile you currently have
```
GaoJianLing
```

I don't know much about name conventions in your country, but maybe if you add spaces, like that:
```
Gao Jian Ling
```
it will pass?

Additionally, I found this article:
https://medium.com/@Chris_Du/apply-for-github-s-student-developer-pack-chinese-students-special-edition-369e27b8a922
which says:
> First, create a github account. Use whatever email address, it doesn’t matter, since github automatically blocks school email requests from China. Then, go here and you’ll find the page as what I show above. Third, fill in the blank out there.
>
> Pay attention to the VERIFY ACADEMIC STATUS choice. Please do remember to choose ‘I don’t have a school-issued email’, or you’ll be rejected immediately. Choose a photo of your student id, and wait for the email. It usually takes a week or two if everything goes on well, but in case you are rather impatient, try this out. It may help, but it depends.

So maybe there's some political issue or maybe GitHub simply does not have the deals or sponsors in China. But at least you can try these options.

**Reply by JerryGao818** [reply]
I guess it would be the political reasons somehow. It's a good idea not using my official e-mail. I will try it latter with my physical receipt. It's also a good idea to add spaces to the name in my profile. Thank you for your detailed instructions! It's very kind of you for all the help :)

**Comment by sumitshinde-84** [comment]
@JerryGao818  you have to upload digital copy of your id card  , because github mechanism cannot  scan  captured photo of your id  so try to upload all documents  orignal digital copy instead of caputured photos .

**Comment by JerryGao818** [comment, marked as answer]
Finally I have my verification thanks to all the valuable suggestions!
![5fe1f0209fec78f7534c4e8561e4bd8](https://user-images.githubusercontent.com/81412682/228125882-5d3ec9e7-09ba-4dfd-8309-73231837ae8b.png)

I think there may be two important things:
1. I have occasionally changed my browser and when I process my application, there was a message trying to get my location (never seen before). It may be important to keep the location the same as my school.
2. Using the dated student ID card, as @KleeMatrix mentioned above. I have tried with other document, and they just didn't work.

对于国内学生，希望我的经历能帮到大家：
1. 记得允许浏览器获取当前位置，确保当前位置和自己的学校位置一致
2. 用学生证的照片，我用的是封面和有注册章的那页，中文没有翻译也没关系，其他的包括教育部学籍在线验证报告之类的我都试过不好用。
3. 保持自己profile里面的各种信息和学生证一致。

Thanks again for all the kind help! Hope this can help someone like me :)

**Reply by AJBrownson** [reply]
Congratulations

**Reply by BlondeMarisa** [reply]
我用学生证还是不好用啊，哎

**Comment by Kenny-Lily** [comment]
请问虽然显示已经通过了，但是并没有收到验证邮件咋办啊，想要用COPILOT写代码
